db/1databaseNotes.txt

```diff
@@ -1,24 +1,37 @@
-# DATABASE NOTES:
-(Martin Connor, 6/5/19);
+DATABASE NOTES:
 
+  (Martin Connor, 6/5/19);
+
+**********************************************************************
+
+TESTING:
+
+# SAVE A QUERY PLAN TO A FILE:
+
+psql -d bookings_portals -qAt -f explain.sql > .txt
+
+
+# TEST 'INNER JOIN' QUERY:
+
+EXPLAIN ANALYZE SELECT bookings.from_date,bookings.to_date FROM bookings INNER JOIN listings ON bookings.listing_id=listings.listing_id WHERE bookings.listing_id=3000000;
+
```

Comment:
You can also wrap several lines in triple backticks
\```
bunch of code,
more code
etc.
\```
versus
```
bunch of code,
more code
etc.
```
Makes for really clear documentation!